### ISSUE TITLE
BLE DTM public APIs in Host

### ISSUE THREAD

**Issue by carlescufi** [zephyrproject-rtos/zephyr#4120]
In order to enable and disable DTM from the BLE Host, we need public APIs to send the corresponding HCI commands.

**Comment by jhedberg** [comment]
@carlescufi do we still want to hero this issue open?

**Comment by carlescufi** [comment]
> @carlescufi do we still want to keep this issue open?

Yes, I think it makes sense to keep this as an enhancement in case someone wants to pick it up.

**Comment by nashif** [comment]
This Enhancement is stale with no followup or details about how it would still apply to the current state of Zephyr. If you think it is still valid, please re-open with additional details and your plans to address it.
